smart-contracts/test/test_txSignatureAggregation.js

```diff
@@ -60,9 +62,15 @@ describe("Gas Cost Tests", function () {
       owner,
       userOne,
       userThree,
-      pauser
+      pauser,
+      networkDescriptor
     );
 
+    // Add the token into white list
+    await state.bridgeBank.connect(operator)
```

Comment:
I see that pretty often we have to add these tokens to the eth whitelist. We may consider making a test fixture that wraps another test fixture and just add the tokens to the eth whitelist.